gocwebtemplate-core/gocwebtemplate-core-base/src/main/java/goc/webtemplate/BannerLink.java

```diff
@@ -0,0 +1,50 @@
+package goc.webtemplate;
+
+public class BannerLink {
```

Comment:
BannerLink should inherit from Link :)  (and not redefine the common stuff)

Reply:
done!